src/z3c/rml/page.py

```diff
@@ -13,65 +13,89 @@
 ##############################################################################
 """Page Drawing Related Element Processing
 """
-import six
+import io
+
 from z3c.rml import attr, directive, interfaces
 
 try:
-    import PyPDF2
-    from PyPDF2.generic import NameObject
+    import pikepdf
+    from pikepdf import Object
 except ImportError:
     # We don't want to require pyPdf, if you do not want to use the features
```

Comment:
Change comment from `pyPdf` to `pikePdf`.

Reply:
done